buddyboss/buddyboss-platform
# Fix # 1810 : Irrelevant names are showing when searching a user in the private messaging

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1810.

### How to test the changes in this Pull Request:

**To Reproduce (in demo site)** please see: https://cutt.ly/LhrqfeZ
Steps to reproduce the behavior:
1. Create another profile field like "my best friend".
2. Edit some users and set "XXXX" in their "my best friend" profile field.
3. Now, go to private messaging and try to search for "XXX" as you recipient.


### Proof Screenshots or Video

https://www.loom.com/share/8862d078681f4313bd7289cfb290c2e0

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
